cmds/dhclient/dhcp6client/dhcp6client_test.go

```diff
@@ -0,0 +1,329 @@
+// Command dhcp6d is an example DHCPv6 server.  It can only assign a
```

Comment:
You can make a file called dhcp6server_test.go that has not tests in it but has this helper code in it. Then it will compile but we can make sure it is separate and has its own copyright.